history/characters/1_azerothian.txt

```diff
@@ -120,6 +120,7 @@
 		add_spouse=482			# Tiffin
 		effect = {
 			add_lover=482			# Tiffin
+			c_5816 ={ add_friend=ROOT } # Reginald Windsor
```

Comment:
It also works and looks better in my opinion.
```suggestion
			add_friend=5816 # Reginald Windsor
```

Reply:
Ok, just went with how the wiki does it. Done